apps/dapp-console-api/src/models/apps.ts

```diff
@@ -0,0 +1,22 @@
+import { integer, pgTable, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+import { entities } from './entities'
+
+enum AppState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+}
+
+export const apps = pgTable('apps', {
```

Comment:
i don't believe postgres will create an index for foregin keys which is what the entity_id would be considered we might need to manually add an index for it

Reply:
good call, just added the index